bitnami/kube-prometheus/Chart.yaml

```diff
@@ -35,4 +35,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/prometheus
   - https://github.com/bitnami/containers/tree/main/bitnami/alertmanager
   - https://github.com/prometheus-operator/kube-prometheus
-version: 8.1.12
+version: 8.1.13
```

Comment:
Since this is adding a new feature, could you increase the minor version instead? (e.g. to `8.2.0`)

Reply:
Sure thing - I figured this was 50/50 bugfix/feature.